test/test_bulk_actions.py

```diff
@@ -237,3 +249,26 @@ def test_clear_incident_in_metadata(metadata):
     assert endpoints["fiberlink"][0]["events"][0]["is_up"] is True
     assert endpoints["fiberlink"][0]["events"][0]["clear_time"] == clear_time
     assert endpoints["fiberlink"][0]["events"][0]["properties"]["status"] == "Clear"
+
+
+@pytest.mark.django_db
+@patch("src.geant_argus.geant_argus.incidents.bulk_actions.clear_alarm")
+def test_bulk_clear_incidents(mock_clear_alarm, default_user):
+    # Mock data
+    actor = default_user
+    qs = [MagicMock(metadata={"status": "ACTIVE", "endpoints": {}})]
+    data = {"timestamp": timezone.now()}
+    clear_time = data["timestamp"].replace(tzinfo=None).isoformat()
+
+    # Mock clear_alarm to return True
+    mock_clear_alarm.return_value = True
```

Comment:
This comment just describes what the line below does, which can also be learned from just reading that line. The comment does not really add any value, and you can better just remove it